factions/ind_me_terrorists.hpp

```diff
@@ -1,119 +1,138 @@
+			addItemsToVest[] = {
+				LIST_3("rhs_mag_9x18_8_57N181S")
+			};
+		};
 
-        //autorifleman
-        class Soldier_AR_F: Soldier_F {
-            primaryWeapon = "rhs_weap_akm";
-            primaryWeaponMagazine = "gm_75Rnd_762x39mm_AP_7N23_ak47_blk";
-            addItemsToUniform[] = {
-                LIST_3("gm_75Rnd_762x39mm_AP_7N23_ak47_blk")
-            };
-        };
+		//autorifleman
+		class Soldier_AR_F: Soldier_F {
+			primaryWeapon = "rhs_weap_akm";
+			primaryWeaponMagazine = "gm_75Rnd_762x39mm_AP_7N23_ak47_blk";
+			addItemsToUniform[] = {
+				LIST_3("gm_75Rnd_762x39mm_AP_7N23_ak47_blk")
+			};
+		};
 
-        //light AT
-        class Soldier_LAT_F: Soldier_F {
-            secondaryWeapon = "rhs_weap_rpg26";
-        };
+		//light AT
+		class Soldier_LAT_F: Soldier_F {
+			secondaryWeapon = "rhs_weap_rpg26";
+		};
 
-        //ammo bearer
-        class Soldier_A_F: Soldier_F {
-            vest = "LOP_6sh46";
-            addItemsToVest[] = {
-                LIST_4("gm_75Rnd_762x39mm_AP_7N23_ak47_blk")
-            };
-        };
+		//ammo bearer
+		class Soldier_A_F: Soldier_F {
+			vest = "LOP_6sh46";
+			addItemsToVest[] = {
+				LIST_4("gm_75Rnd_762x39mm_AP_7N23_ak47_blk")
+			};
+		};
 
-        //assistant autorifleman
-        class Soldier_AAR_F: Soldier_F {
-            vest = "LOP_6sh46";
-            addItemsToVest[] = {
-                LIST_4("gm_75Rnd_762x39mm_AP_7N23_ak47_blk")
-            };
-        };
+		//assistant autorifleman
+		class Soldier_AAR_F: Soldier_F {
+			vest = "LOP_6sh46";
+			addItemsToVest[] = {
+				LIST_4("gm_75Rnd_762x39mm_AP_7N23_ak47_blk")
+			};
+		};
 
-        //medic
-        class Medic_F: Soldier_F {
-            vest = "LOP_6sh46";
-            headgear = "LOP_H_Pakol";
-            goggles = "G_Bandanna_khk";
-            addItemsToVest[] = {
-                LIST_20("ACE_fieldDressing"),
-                LIST_15("ACE_morphine"),
-                LIST_15("ACE_epinephrine"),
-                LIST_2("ACE_bloodIV_250")
-            };
-        };
+		//medic
+		class Medic_F: Soldier_F {
+			vest = "LOP_6sh46";
+			headgear = "LOP_H_Pakol";
+			goggles = "G_Bandanna_khk";
+			addItemsToVest[] = {
+				LIST_20("ACE_fieldDressing"),
+				LIST_15("ACE_morphine"),
+				LIST_15("ACE_epinephrine"),
+				LIST_2("ACE_bloodIV_250")
+			};
+		};
 
-        //squad leader
-        class Soldier_SL_F: Soldier_F {
-            binoculars = "Binocular";
-        };
+		class Soldier_TL_F {
+			binoculars = "Binocular";
+			headgear = "LOP_H_Pakol";
+			goggles = "rhs_scarf";
+			primaryWeapon = "rhs_weap_savz61";
+			primaryWeaponMagazine = "rhsgref_20rnd_765x17_vz61";
+			addItemsToUniform[] = {
+				"rhs_mag_rgo",
+				LIST_3("rhs_mag_rdg2_white")
+			};
+			addItemsToVest[] = {
+				LIST_6("rhsgref_20rnd_765x17_vz61")
+			};
+		};
+		class Soldier_SL_F: Soldier_TL_F {
+			uniform = "LOP_U_AM_Fatigue_02";
+			headgear = "LOP_H_Turban_mask";
+			secondaryWeapon = "gm_wz78_blk";
```

Comment:
Needs to be `handgunWeapon` and `handgunWeaponMagazine` instead of `secondaryWeapon`.